libMBIN/Source/Common/MathHelper.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace libMBIN.Source.Common
+{
+    internal static class MathHelper
+    {
+        /// <summary>
+        /// Clamps a value to be equal to or between the minimum and maximum.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <param name="minimum"></param>
+        /// <param name="maximum"></param>
+        /// <returns></returns>
+        public static int Clamp(int value, int minimum, int maximum)
+        {
+            if (value < minimum)
+                value = minimum;
+            if (value > maximum)
+                value = maximum;
+
+            return value;
+        }
+
+        /// <summary>
+        /// Clamps a value to be equal to or between the minimum and maximum.
+        /// </summary>
+        /// <param name="value"></param>
+        /// <param name="minimum"></param>
+        /// <param name="maximum"></param>
+        /// <returns></returns>
+        public static float Clampf(float value, float minimum, float maximum)
```

Comment:
why is this called `Clampf` while all the others are just called `Clamp`? Surely cos it's an overloaded method you just want it to be called `Clamp`?